.github/workflows/deploy-cluster.yml

```diff
@@ -29,15 +29,14 @@ jobs:
 
   deploy:
     name: Deploy Cluster
-    if: github.ref == 'refs/heads/main'
```

Comment:
Shouldn't we keep this condition?

Reply:
Yes, we should -- I removed for testing -- need to put it back inn.